src/webgl-preview.ts

```diff
@@ -289,7 +302,7 @@ export class WebGLPreview {
     this.render();
   }
 
-  render(): void {
+  initScene() {
```

Comment:
I agree with splitting out and explosing `initScene`. Not a breaking change because `render` still uses it internally. Clients don't have to explicitly call it, but they now can if they need to.

Reply:
exposing it was an accident :-) I want to expose only what is required. Especially bc I think the methods initScene and clear still need to be matured